content/blog/2021-03-31-march-21-community-gems.md

```diff
@@ -0,0 +1,102 @@
+---
+title: March '21 Community Gems
+date: 2021-03-31
+description: |
+  A roundup of technical Q&A's from the DVC community. 
+  This month: remote storage integration, hyperparameter tuning,
+  best practices for managing experiments and more.
+descriptionLong: |
+  A roundup of technical Q&A's from the DVC community. 
+  This month: remote storage integration, hyperparameter tuning,
+  best practices for managing experiments and more.
+picture: 2021-03-31/gems-cover.png
+author: jeny_defigueiredo
+commentsUrl: https://discuss.dvc.org/t/march-2021-community-gems/708
+tags:
+  - Discord
+  - Gems
+  - Pipelines
+  - Remote Storage
+---
+
+## DVC questions
+
+### [Q: Will DVC work with <my remote cloud storage of choice?>](https://discord.com/channels/485586884165107732/563406153334128681/821493606770409493)
+
+We recently had questions about this, specifically regarding Huawei Cloud and
+Backblaze B2 Storage. The answer is any cloud storage that has an s3 interface
+will work with DVC and both of the aforementioned do!  
+[Learn more about remote storage integrations here.](https://dvc.org/doc/command-reference/remote)
```

Comment:
let's specifically mention that "S3 Compatible" expandable is of the interest on this page for this question